src/pages/index.js

```diff
@@ -17,86 +17,235 @@ import {
-  ".elements"
-);
-
-firstCards.renderItems();
+const api = new Api({
+  apiAddress: "https://around-api.en.tripleten-services.com/v1",
+  apiHeaders: {
+    authorization: "34ba410c-a4f9-4189-8d1e-4545749c88e4",
+    "Content-Type": "application/json",
+  },
+});
 
 const newUserInfo = new UserInfo({
   userNameSelector: ".info__name",
   userJobSelector: ".info__job-title",
+  profilePictureSelector: ".profile__circle",
 });
 
-const profileModal = new PopupWithForm({
-  popupSelector: ".modal_type_profile",
-  handleFormSubmit: (formData) => {
-    console.log(formData.name);
-    console.log(formData.job);
-    newUserInfo.setUserInfo({ name: formData.name, job: formData.job });
-    profileModal.close();
-  },
+api
+  .loadPageResults()
+  .then(([cards, userData]) => {
+    cardArea = new Section({ items: cards, renderer: createCard }, ".elements");
+    cardArea.renderItems();
+    console.log(userData);
+    newUserInfo.setUserInfo({
+      name: userData.name,
+      job: userData.about,
+    });
+    newUserInfo.setUserPicture({ avatar: userData.avatar });
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+//Create cards and handle their like and delete buttons
+
+function createCard(item) {
+  const card = new Card(
+    item,
+    "#localeCard",
+    openModalImage,
+    handleDeleteClick,
+    handleLike
+  );
+  return card.generateCard();
+}
+
+const cardPopOut = new PopupWithImage({
+  popupSelector: ".modal_type_image-pop-out",
+  imageSelector: ".card__image_option_pop-out",
 });
 
-const newImageModal = new PopupWithForm({
-  popupSelector: ".modal_type_new-image",
-  handleFormSubmit: (newData) => {
-    const addedCard = createCard(newData);
-    firstCards.addItem(addedCard);
-    modalImageForm.reset();
-    newImageValidator.toggleButtonState();
-    newImageModal.close();
+cardPopOut.setEventListeners();
+
+const deleteCardModal = new PopupWithDelete({
+  popupSelector: ".modal_type_delete-image",
+  handleBtnClick: () => {
+    api
+      .deleteCard(cardToDelete.data._id)
+      .then(cardToDelete.deleteCard())
+      .then(deleteCardModal.close())
+      .catch((err) => {
+        console.error(err);
+      })
+      .finally(deleteCardModal.renderSaving(false));
   },
+  buttonText: "Yes",
 });
 
-function openModalImage(card) {
-  cardPopOut.open(card.data.name, card.data.link);
+deleteCardModal.setEventListeners();
+
+function handleDeleteClick(card) {
+  console.log(card);
+  deleteCardModal.open();
+  cardToDelete = card;
+  return cardToDelete;
+}
+
+function handleLike(card) {
+  if (card.data.isLiked == false) {
+    api
+      .addLike(card.data._id)
+      .then((card.data.isLiked = true))
+      .then(card._cardHeart.classList.toggle("card__heart-option-liked"))
+      .catch((err) => {
+        console.error(err);
+      });
+  } else {
+    api
+      .removeLike(card.data._id)
+      .then((card.data.isLiked = false))
+      .then(card._cardHeart.classList.toggle("card__heart-option-liked"))
```

Comment:
`_cardHeart`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.